lib/shopt.sh

```diff
@@ -27,7 +27,15 @@ shopt -s nocaseglob;
 ## SMARTER TAB-COMPLETION (Readline bindings) ##
 
 # Perform file completion in a case insensitive fashion
-bind "set completion-ignore-case on"
+if [ ! -z "$CASE_SENSITIVE" ]; then
+	if [[ $CASE_SENSITIVE == "true" ]];then
+		bind "set completion-ignore-case off"
+	else
+		bind "set completion-ignore-case on"
+	fi
+else
+	bind "set completion-ignore-case on"
+fi
```

Comment:
Two branches are the same, please rewrite this

Reply:
I think the logic is fine. There's 2 branches that are the same, but that's just circumstantial (i.e. one corresponds to the "default" case and the other one for when the case sensitivity is explicitly enabled).

Maybe it can be slightly rewritten to be more clear?
```bash
if [[ ${CASE_SENSITIVE} == false ]]; then
  bind "set completion-ignore-case on"
elif [[ ${CASE_SENSITIVE} == true ]]; then
  bind "set completion-ignore-case off"
else
  # By default, case sensitivity is disabled.
  bind "set completion-ignore-case on"
fi
```